lib/client/client.go

```diff
@@ -290,7 +254,7 @@ func (proxy *ProxyClient) IssueUserCertsWithMFA(ctx context.Context, params Reis
 			// Probably talking to an older server, use the old non-MFA endpoint.
 			log.WithError(err).Debug("Auth server does not implement IsMFARequired.")
 			// SSH certs can be used without reissuing.
-			if params.usage() == proto.UserCertsRequest_SSH {
+			if params.usage() == proto.UserCertsRequest_SSH && key.Cert != nil {
```

Comment:
Since SSH certs are now cluster-specific they are not necessarily part of every `Key` structure. (The "core" parts are now just the RSA private/public key and the Auth TLS certificate.) Therefore if the MFA request expects the SSH cert to be present and it is not part of the present `key`, we have to go through the full reissue procedure instead of just returning `key` right away.